var/spack/repos/builtin/packages/picsarlite/package.py

```diff
@@ -0,0 +1,137 @@
+##############################################################################
+# Copyright (c) 2018, Los Alamos National Security, LLC.
+# Produced at the Los Alamos National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Picsarlite(MakefilePackage):
+    """PICSARlite is a self-contained proxy that adequately portrays the
+       computational loads and dataflow of more complex PIC codes.
+    """
+
+    tags = ['proxy-app', 'ecp-proxy-app']
+
+    homepage = "http://picsar.net"
+    git      = "https://bitbucket.org/berkeleylab/picsar.git"
+
+    version('develop', branch='PICSARlite')
+    version('0.1', tag='PICSARlite-0.1')
+
+    variant('prod', default=False, 
+            description='Production mode (without FFTW)')
+    variant('prod_spectral', default=False, 
```

Comment:
Installing with that variant fails for me since the dependency to `FFTW` was not declared. Add something like:
```py
depends_on('ffwt@min_version: +mpi', when='+prod_spectral')
```